1.9/overview/architecture/node-types.md

```diff
@@ -0,0 +1,79 @@
+
+Like Mesos, several DC/OS master node components (e.g. Marathon &amp; Zookeeper) also perform independent leader election. This means that the leaders for different components may be on different master nodes.
+
+### Quorum
+
+To maintain consistency, a quorum (half plus one) of master nodes must be connected at all times.
+
+For example, having three master nodes allows one to be down; having five master nodes allows two to be down, allowing for failure during a rolling update. Additional master nodes can be added for additional risk tolerance.
+
+DC/OS currently only allows specifying the number of master nodes at install time, due mostly to the complexity of changing the quorum and configuration of multiple components with leaders on different nodes.
+
+
+## Agent Nodes
+
+A DC/OS agent node is a node on which user tasks are run.
+
+Agent nodes contain a few DC/OS components, including most notably a Mesos agent process.
+
+Agent nodes can be public or private, depending on agent and network configuration.
+
+### Public Agent Nodes
+
+A public agent node is an agent node that is on a network with have ingress access from outside of the cluster via the cluster’s [infrastructure networking](/docs/1.9/overview/concepts/#infrastructure-network).
+
+The resources on private agent nodes are, by default, configured to only be allocated to tasks that specify the `slave_public` role. The Mesos agents on private agent nodes also have the `public_ip:true` agent attribute to assist in their discovery.
+
+Public agent nodes are used primarily for externally facing reverse proxy load balancers, like Marathon-LB. This provides a [DMZ](https://en.wikipedia.org/wiki/DMZ_(computing)) that decreases the surface area accessible by malicious attackers.
+
+Clusters generally have only a few public agent nodes, because a few load balancers can usually handle proxying to multiple services.
+
+### Private Agent Nodes
+
+A private agent node is an agent node that is on a network that does not have ingress access from outside of the cluster via the cluster’s [infrastructure networking](/docs/1.9/overview/concepts/#infrastructure-network).
+
+The resources on private agent nodes are, by default, configured to allow undifferentiated allocation. More precisely, the resources are given the `*` role, allowing them to be allocated to any task that does not specify a role. For more information, see [Mesos resource roles](http://mesos.apache.org/documentation/latest/roles/).
+
+Because these resources are undifferentiated, most tasks are scheduled on private agent nodes and are thus inaccessible from outside the cluster, decreasing the surface area accessible by malicious attackers. For this reason, clusters are generally comprised of mostly private agent nodes. Likewise, most [Mesosphere Universe](https://dcos.io/docs/1.8/overview/concepts/#mesosphere-universe) packages install by default on private agent nodes.
+
+## See Also
```

Comment:
"See Also" is standard terminology in man pages and wikipedia. Are we using "More Information" elsewhere as precedent?